src/pattern-library/components/patterns/feedback/PopoverPage.tsx

```diff
@@ -0,0 +1,176 @@
+import { Link } from '../../../../components/navigation';
+import Library from '../../Library';
+
+export default function PopoverPage() {
+  return (
+    <Library.Page
+      title="Popover"
+      intro={
+        <>
+          <p>
+            <code>Popover</code> is a floating component that can be shown
+            next/relative to another anchor element.
+          </p>
+        </>
+      }
+    >
+      <Library.Section
+        title="Popover"
+        intro={
+          <>
+            <p>
+              By default, <code>Popover</code> will be displayed under the
+              anchor element, unless there is not enough space below.
+            </p>
+            <p>
+              It will always be at least as big as the anchor element, but it
```

Comment:
```suggestion
              It will always be at least as wide as the anchor element, but it
```

Make it clear which dimension "big" refers to.